lib/search.rb

```diff
@@ -837,8 +837,26 @@ def posts_query(limit, opts = nil)
       end
     else
       data_ranking = <<~SQL
-      TS_RANK_CD(
-        post_search_data.search_data, #{ts_query(weight_filter: weights)}
+      (
+        TS_RANK_CD(
+          post_search_data.search_data,
+          #{ts_query(weight_filter: weights)},
+          0|32
```

Comment:
These magic numbers (flags) need at the very least a comment explaining what they do

Reply:
Hmm these numbers are in the documentation for the function being used here and I would prefer not to have to duplicate the Postgres docs here.

Reply:
I don't mean to copy-paste the postgres doc, but why do you use this combination of flags?